### PR TITLE
Updated Omnibus with RPM dist tag fixes

### PR DESCRIPTION
**Pull request by schisamo**
This PR brings in an updated RPM packager which adds a Fedora [Dist Tag](http://fedoraproject.org/wiki/Packaging:DistTag) (.e.g. `.el5`, `.el6`, `.el7` etc.) to the package name and spec's [Release Tag](http://fedoraproject.org/wiki/Packaging:NamingGuidelines#Release_Tag). This ensures the RPMs more closely match the Fedora packaging guidelines which many users and tools depend on.

Successful CI runs:
http://wilson.ci.chef.co/job/supermarket-trigger-ad_hoc/2/downstreambuildview/

/cc @chef/ociv @chef/chef-visual-interaction @chef/community-engineering-team

### CODE REVIEW

**Comment by someara** [comment]
looks good to me

**Comment by nathenharvey** [comment]
:+1:

File: Gemfile.lock [hunk 1]
```diff
@@ -66,7 +78,7 @@ GEM
     mixlib-cli (1.5.0)
     mixlib-config (2.1.0)
     mixlib-log (1.6.0)
-    mixlib-shellout (1.4.0)
+    mixlib-shellout (1.6.1)
```
Comment (lamont-granquist) [inline Gemfile.lock]: omnibus should really get onto ohai 8.x/mixlib-shellout 2.x

**Comment by lamont-granquist** [comment]
:+1:

**Comment by smurawski** [comment]
:+1:

File: Gemfile.lock (hunk 1 above)
Comment (schisamo) [inline Gemfile.lock]: PRs welcome. :)

**Pull request merged**